libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -2234,7 +2234,7 @@ void WaveTrack::PasteWaveTrackAtSameTempo(
 
     //wxPrintf("paste: we have at least one clip\n");
 
-    const auto clipAtT0 = track.GetIntervalAtTime(t0);
+    const auto clipAtT0 = track.GetWideClipAtTime(t0);
```

Comment:
I wonder why `GetIntervalAtTime` isn't sufficient.
In fact, looking at the `GetWideClipAtTime` implementation, I see logic that might have been relevant before the 2-of-6 PR, but which now seems futile to me. More about this in a follow-up comment closer to that implementation.